diffgraph/processing_modes/tree_sitter_dependency.py

```
@@ -0,0 +1,1185 @@
+
+            # Parse pre snapshot
+            pre_components = []
+            if pre_content:
+                try:
+                    parser = self._get_parser(language)
+                    pre_tree = parser.parse(pre_content.encode("utf-8"))
+                    pre_components = self._extract_components(
+                        language, pre_tree, pre_content.encode("utf-8"), file_path
+                    )
+                except Exception as e:
+                    pass  # Treat parse failure as empty pre-snapshot
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)